src/components/Global/ErrorDisplay/index.tsx

```diff
@@ -12,7 +13,7 @@ const ErrorDisplay = ({
   return (
     <div className={styles.center}>
       <div className={styles.contentWrapper}>
-        <div className={styles.logo}>
+        <div className={styles.logo} onClick={ () => Router.push('/') } >
           <Logo showMark={true} showType={true} />
```

Comment:
The new Logo component in main has a prop you can pass to make it clickable and returns to home